apps/test/unit/applab/designModeTest.js

```diff
@@ -1,5 +1,10 @@
+import sinon from 'sinon';
```

Comment:
Given how much was added to this file by testing themes (4x file size increase), can you move themes to a different file?